controller_report_xls/controllers/main.py

```diff
@@ -38,6 +38,96 @@
 _logger = logging.getLogger(__name__)
 
 
+def is_float(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_integer(value):
+    try:
+        int(value)
+        return True
+    except ValueError:
+        return False
```

Comment:
You have to realize that sometimes numbers are formatted as

> u'-7,777,777.77' 
or 
> u'-7.777.777,77' 

this string should regarded as a number with your regex that won't happen

What we have intended is to unformat string into integer/float
reverse the process performed by `formatLang`

Which is complained here 

https://www.python.org/dev/peps/pep-0378/

> Some commenters do not like the idea of format strings at all and find them to be unreadable. Suggested alternatives include the COBOL style PICTURE approach or a convenience function with keyword arguments for every possible combination.